CMakeLists.txt

```diff
@@ -651,8 +651,8 @@ kratos_python_install(${INSTALL_PYTHON_USING_LINKS} "${KRATOS_SOURCE_DIR}/kratos
 kratos_python_install(${INSTALL_PYTHON_USING_LINKS} "${KRATOS_SOURCE_DIR}/kratos/python_interface/python_registry_utilities.py" "KratosMultiphysics/python_registry_utilities.py" )
 
 # Install the libraries in the libs folder
-install(FILES ${Boost_LIBRARIES} DESTINATION libs)
-install(FILES ${EXTRA_INSTALL_LIBS} DESTINATION libs)
```

Comment:
@markelov208 Please do not change this.

Libs needs to be deleted and repopulated at every compilation to avoid mismatching versions of the same library. This change is intentional.

Reply:
@roigcarlo , thank you for letting me know. Currently, the libs folder is populated and then deleted. 
[39/40] Install the project...
-- Install configuration: "Debug"
-- Deleting: D:/Checkouts2/KratosProjects/alpha/bin/Debug/KratosMultiphysics
-- Deleting: D:/Checkouts2/KratosProjects/alpha/bin/Debug/libs

Build finished

Please could you let me know how to change the order in cmake file? Thank you.

Reply:
The log its a little bit misleading, its the other way around, libs folders gets deleted and then populated if the compilation goes well

Reply:
Unfortunately, this is exactly what I have on my laptop: files are copied into libs folder, then libs is deleted. ;) I have reverted libs and put REMOVE_INSTALL_DIRECTORIES OFF.

Reply:
Mmm that's quite odd. In any case, its important to maintain the name `libs`, so if it fails as last resource we can comment the lines that remove the contents of the folder, but changing the name will effectively break the release and RPATH so that cannot be changed